RecoEgamma/EgammaPhotonProducers/src/PhotonProducer.cc

```diff
@@ -1,483 +1,505 @@
 
-PhotonProducer::PhotonProducer(const edm::ParameterSet& config) :
-
-  conf_(config)
-{
+PhotonProducer::PhotonProducer(const edm::ParameterSet& config)
+    :
 
+      conf_(config) {
   // use onfiguration file to setup input/output collection names
 
-  photonCoreProducer_   =
-    consumes<reco::PhotonCoreCollection>(conf_.getParameter<edm::InputTag>("photonCoreProducer"));
-  barrelEcalHits_   =
-    consumes<EcalRecHitCollection>(conf_.getParameter<edm::InputTag>("barrelEcalHits"));
-  endcapEcalHits_   =
-    consumes<EcalRecHitCollection>(conf_.getParameter<edm::InputTag>("endcapEcalHits"));
-  vertexProducer_   =
-    consumes<reco::VertexCollection>(conf_.getParameter<edm::InputTag>("primaryVertexProducer"));
-  hcalTowers_ =
-    consumes<CaloTowerCollection>(conf_.getParameter<edm::InputTag>("hcalTowers"));
-  hOverEConeSize_   = conf_.getParameter<double>("hOverEConeSize");
-  highEt_        = conf_.getParameter<double>("highEt");
+  photonCoreProducer_ = consumes<reco::PhotonCoreCollection>(
+      conf_.getParameter<edm::InputTag>("photonCoreProducer"));
+  barrelEcalHits_ = consumes<EcalRecHitCollection>(
+      conf_.getParameter<edm::InputTag>("barrelEcalHits"));
+  endcapEcalHits_ = consumes<EcalRecHitCollection>(
+      conf_.getParameter<edm::InputTag>("endcapEcalHits"));
+  vertexProducer_ = consumes<reco::VertexCollection>(
+      conf_.getParameter<edm::InputTag>("primaryVertexProducer"));
+  hcalTowers_ = consumes<CaloTowerCollection>(
+      conf_.getParameter<edm::InputTag>("hcalTowers"));
+  hOverEConeSize_ = conf_.getParameter<double>("hOverEConeSize");
+  highEt_ = conf_.getParameter<double>("highEt");
   // R9 value to decide converted/unconverted
-  minR9Barrel_        = conf_.getParameter<double>("minR9Barrel");
-  minR9Endcap_        = conf_.getParameter<double>("minR9Endcap");
-  usePrimaryVertex_   = conf_.getParameter<bool>("usePrimaryVertex");
-  runMIPTagger_       = conf_.getParameter<bool>("runMIPTagger");
+  minR9Barrel_ = conf_.getParameter<double>("minR9Barrel");
+  minR9Endcap_ = conf_.getParameter<double>("minR9Endcap");
+  usePrimaryVertex_ = conf_.getParameter<bool>("usePrimaryVertex");
+  runMIPTagger_ = conf_.getParameter<bool>("runMIPTagger");
 
-  candidateP4type_ = config.getParameter<std::string>("candidateP4type") ;
+  candidateP4type_ = config.getParameter<std::string>("candidateP4type");
 
   edm::ParameterSet posCalcParameters =
-    config.getParameter<edm::ParameterSet>("posCalcParameters");
+      config.getParameter<edm::ParameterSet>("posCalcParameters");
   posCalculator_ = PositionCalc(posCalcParameters);
 
-
-  //AA
-  //Flags and Severities to be excluded from photon calculations
+  // AA
+  // Flags and Severities to be excluded from photon calculations
   const std::vector<std::string> flagnamesEB =
-    config.getParameter<std::vector<std::string> >("RecHitFlagToBeExcludedEB");
+      config.getParameter<std::vector<std::string> >(
+          "RecHitFlagToBeExcludedEB");
```

Comment:
line breaks seem unnecessary.
also, this format update is not consistent within other updates in this file: e.g. consumes were pushed to the same line as "=" up to "("